src/io.py

```diff
@@ -34,4 +34,4 @@ def parse_input() -> (list, int, bool, str, str, str):
 
 def format_output(output_strings: dict) -> None:
     for name, value in output_strings.items():
-        print(f'::set-output name={name}::{value}')
+        print(f'"{name}={value}" >> $GITHUB_OUTPUT')
```

Comment:
Mm not sure this is going to work, this will just print this to stdout but won't actually write to that file at `$GITHUB_OUTPUT`. You're probably gonna need to open the file for writes and append that string to the end instead.

Reply:
You're right. I committed another suggestion with os.system. Wdyt ?

Reply:
Just tested on a private repo, the output is set 👌 :
![image](https://user-images.githubusercontent.com/1169844/200558643-9734ab62-c4da-4bf1-bdc7-b4718ee08643.png)

Reply:
I'm not sure how to interpret "$GITHUB_OUTPUT" in Python (it's definetly not the file name, it has to be replace by its value. Happy to do the file opening/writing part, but I guess you'll be faster than me. Thanks

Reply:
you can retrieve the value of the environment variable (eg `file_path = getenv('GITHUB_OUTPUT')`, then use python's filesystem functions to append to it. For instance:

```
file_path = getenv('GITHUB_OUTPUT')
with open(file_path, "a") as gh_output:
    gh_output.write("foo=bar")
```

Reply:
Thanks. Pushed a clean history version.

Reply:
This is it. Gonna run some tests but looks like it's correct.